resources/views/settings/edit.php

```diff
@@ -88,9 +88,9 @@
                     <option value="Web View" <?php echo ( $existing_data['type'] === 'Web View' ) ? 'selected' : ''; ?>>
                         Web View
                     </option>
-                    <option value="Custom" <?php echo ( $existing_data['type'] === 'Custom' ) ? 'selected' : ''; ?>>
+                    <!--<option value="Custom" <?php /*echo ( $existing_data['type'] === 'Custom' ) ? 'selected' : ''; */?>>
                         Custom
```

Comment:
Custom should be an option in the edit screen, but you won't be able to edit the field since it's read-only.